utility/mm2_update-master-directory-list

```diff
@@ -581,17 +581,23 @@ def sync_category_directories(
         d = category.directory_cache[relativeDName]
         D = mirrormanager2.lib.get_directory_by_id(session, d.id)
 
-        target = None
-        if 'repomd.xml' in value['files']:
+        if value['isRepository']:
             target = 'repomd.xml'
-        elif 'summary' in value['files']:
+        elif value['isAtomic']:
             target = 'summary'
         else:
-            continue
+            target = None
 
         if value['isRepository'] or value['isAtomic']:
             make_repository(session, D, relativeDName, category, target)
 
+        if 'repomd.xml' in value['files']:
+            target = 'repomd.xml'
+        elif 'summary' in value['files']:
+            target = 'summary'
+        else:
+            continue
+
```

Comment:
I remember re-writing/simplifying this part in the UMDL2

I'm not fan of this change but since UMDL1's days are counted let's do this :)